### DISCUSSION TITLE
Friendship on Nostr: here you can publish your pubkey with a little presentation

### DISCUSSION THREAD

**Discussion by serrq** [KoalaSat/nostros#268] [General]
Hi, I am serrq (fantasy name) from Italy. I am not a developer. I studied philosophy and my posts on Nostr mostly are about philosophy and literature. Here for friendship with developers and other categories. 
I am still learning English. 👋

nostr:

npub1cy3eve2u92ufwe9qthwttmmqqrwfschmr77t0nxxkgx5arw3rxtqjmr0zm

**Comment by KoalaSat** [comment]
Hi @serrq ! Great idea! this is mine: `645681b9d067b1a362c4bee8ddff987d2466d49905c26cb8fec5e6fb73af5c84`
